src/main.cpp

```diff
@@ -2468,6 +2465,13 @@ void vcMain_RenderSceneWindow(vcState *pProgramState)
     vcCamera_UpdateMatrices(pProgramState->geozone, &pProgramState->pActiveViewport->camera, pProgramState->settings.camera, udFloat2::create(pProgramState->settings.viewports[0].resolution));
   }
 
+  if (pProgramState->exportVideo && pProgramState->settings.viewports[0].resolution != pProgramState->exportVideoResolution)
+  {
+    pProgramState->settings.viewports[0].resolution = pProgramState->exportVideoResolution;
+    vcRender_ResizeScene(pProgramState, pProgramState->pActiveViewport->pRenderContext, pProgramState->exportVideoResolution.x, pProgramState->exportVideoResolution.y);
+    vcCamera_UpdateMatrices(pProgramState->geozone, &pProgramState->pActiveViewport->camera, pProgramState->settings.camera, udFloat2::create(pProgramState->settings.viewports[0].resolution));
```

Comment:
The screenshot system binds the default framebuffer, why doesn't this?

Reply:
I don't think the bind actually accomplishes anything. I asked paul about this, but he hasnt responded yet